cmd/tailscale/main.go

```diff
@@ -58,6 +58,8 @@ type App struct {
 	store             *stateStore
 	logIDPublicAtomic atomic.Pointer[logid.PublicID]
 
+	localAPIClient *localapiclient.LocalAPIClient
```

Comment:
We can keep one instance of localAPIClient on the app for use in methods like getBugReportID().